docs/graphics/home-page/HeroGraphic.js

```diff
@@ -1,7 +1,7 @@
 // @flow strict
 import { type Node } from 'react';
 
-export default function Hero(): Node {
+export default function Hero({ animate = true }: {| animate?: boolean |}): Node {
```

Comment:
nit: I would think animation wouldn't be the default state…like, here's a Hero, and here's a Hero that's ✨animated✨
and from a "reading the code" perspective, it's a bit unintuitive that the Hero is always animated unless you turn that off — like I would expect `<Hero />` and `<Hero animate />` vs `<Hero />` and `<Hero animate={false} />` (I feel like boolean props should always default to false, that `boolProp={false}` is a code smell)

also, super nit: do you mind renaming this `HeroGraphic` so it matches the file name?

Reply:
This is the home page hero, which we would want to animate always, but we're only turning it off now, because there's a new animation on top. So once we remove the banner, it'd just go back to <Hero/>